assignment1.txt

```diff
@@ -0,0 +1,14 @@
+Madeleine Charity
+ 
+ 1) What is a compiler?
+    
+A compiler converts code in one language into another language. Our compiler (gcc) converts C into assembly.
```

Comment:
right, more specifically, a compiler takes your code written in C, removes the whitespace, places this into a parse tree, optimizes it, and then takes that parse tree and translates it into assembly code.